cli/scripts/mindcli-ws

```diff
@@ -139,6 +139,21 @@ echo_progress() {
 	echo -e "$(printf "${status} %.0f" $(echo -e "${progress} * 100" | bc -l))%"
 }
 
+check_auth() {
```

Comment:
This function seems not only check_auth, I think we should think of a better name.